blog/2021-07-13-tdex-terraform-automation.md

```diff
@@ -0,0 +1,67 @@
+---
+slug: tdex-terraform-automation
+title: Automate TDex deamon with terraform
+author: Alexander K
+author_title: Maintainer of TDEX Network
+tags: [terraform, automation, tdex]
+---
+
+Configure and run TDex box easily using Terraform automation. 
+
+<!--truncate-->
+
+
+As a first step it is required toobtain aws access and secret keys. Best practice is to create new user in AWS IAM. Proceed with opening AWS IAM users page, and click on Add user. Provide user with name, and below for the Access type select Programmatic access. Next, make sure that your AWS account(access key) has all required privileges to create EC2 instances and S3 access. 
```

Comment:
Please would be nice to have screenshot of AWS console at least, ie. how to get secret/access key etc..

Better to describe each step with bullet points and separate sections for better readability